extension/meminfo.c

```diff
@@ -448,12 +454,22 @@ void meminfo_zval_dump(php_stream * stream, char * frame_label, char * symbol_na
     php_stream_printf(stream TSRMLS_CC, "        \"type\" : \"%s\",\n", zend_get_type_by_const(Z_TYPE_P(zv)));
     php_stream_printf(stream TSRMLS_CC, "        \"size\" : \"%ld\",\n", meminfo_get_element_size(zv));
 
+    php_json_encode(&buf, zv, (int)0);
```

Comment:
I'm afraid this only works if the JSON extension is loaded, which is not guaranteed.